js/indexMovies.js

```diff
@@ -1,39 +1,48 @@
-
-$( document ).ready(function() {
-  axios.get('https://api.themoviedb.org/3/discover/movie?api_key=ca5d667528ca51e527d9e4f7830d97d2&language=en-US&sort_by=popularity.desc&include_adult=false')
-  .then((response) =>{
-    let movies = response.data.results;
-    console.log(response);
-    let output ='';
-    $.each(movies, (index, movie) =>{
-      output += `
+$(document).ready(function () {
+  axios
+    .get(
+      'https://api.themoviedb.org/3/discover/movie?api_key=ca5d667528ca51e527d9e4f7830d97d2&language=en-US&sort_by=popularity.desc&include_adult=false'
+    )
+    .then(response => {
+      let movies = response.data.results
```

Comment:
You can also add checks to see if results exists or is defined then only the code should proceed further.